egs/wsj/s5/utils/validate_disambig_sym_file.pl

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env perl
+
+# Copyright 2016 FAU Erlangen (Author: Axel Horndasch)
+# Apache 2.0.
+#
+# Concept: Dan Povey
+
+use strict;
+use warnings;
+use Getopt::Long;
+
+my $Usage = <<EOU;
+Usage:  validate_disambig_sym_file.pl [options] disambig_syms.txt
+
+This scripts checks if the entries of a file containing disambiguation symbols
+(word or phone level) are all valid. To be valid the symbols
+- must start with the hash mark '#',
+- must not contain any whitespace,
+- must not be equal to '#-1' (disallowed because it is used internally in some
+  FST stuff).
+
+In case the option '--extra-disambig-syms' is used with 'true', the symbols must
+also be non-numeric (to avoid overlap with the automatically generated symbols).
+
+Allowed options:
+  --extra-disambig-syms  : Indicate extra symbols  (boolean,  default = "false")
```

Comment:
I think it would be clearer and more consistent with the bash scripts to write this as follows (and change the name):
 ```   --allow-numeric    (true|false)    :   Default true.  If false, disallow numeric disambiguation
                                                                    symbols like #0, #1 and so on.
 ```